scripts/add.tokens.erc20.mjs

```diff
@@ -0,0 +1,264 @@
+import { load } from 'cheerio';
+import dotenv from 'dotenv';
+import { ethers } from 'ethers';
+import fs from 'fs/promises';
+import fetch from 'node-fetch';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import path from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const rootPath = resolve(__dirname, '../');
+
+const envFile = process.env.NODE_ENV === 'production' ? '.env' : '.env.development';
+const envPath = resolve(rootPath, envFile);
+dotenv.config({ path: envPath });
```

Comment:
Can't we use `config({ path: `.env.${ENV}` });` as in other scripts?

Reply:
done